scripts/core/editor3/reducers/tests/suggestion_style.spec.jsx

```diff
@@ -0,0 +1,244 @@
+import * as Highlights from '../../helpers/highlights';
+import * as Setup from './suggestion_setup';
+
+describe('editor3.reducers.suggestion.CREATE_CHANGE_STYLE_SUGGESTION', () => {
+    it('CREATE_CHANGE_STYLE_SUGGESTION - set BOLD suggestion', () => {
+        const rawContent = {
+            blocks: [
+                {key: '4vu4i', text: 'paragraph1'},
+                {key: '9d99u', text: 'paragraph2'},
+            ],
+            entityMap: {},
+        };
+        const date = new Date();
+        let editorState = Setup.getInitialEditorState(rawContent);
+
+        editorState = Setup.applySelection(editorState, 0, 4, 1, 6);
+        editorState = Setup.addStyleSuggestion(editorState, 'BOLD', date);
+
+        const selection = editorState.getSelection();
+        const content = editorState.getCurrentContent();
+
+        expect(content.getPlainText()).toEqual('paragraph1\nparagraph2');
+        expect(selection.getStartOffset()).toEqual(4);
+        expect(selection.getStartKey()).toEqual(content.getFirstBlock().getKey());
+        expect(selection.getEndOffset()).toEqual(6);
+        expect(selection.getEndKey()).toEqual(content.getLastBlock().getKey());
+
+        let block = content.getFirstBlock();
+
+        for (let i = 0; i < 10; i++) {
+            if (i < 4) {
+                expect(block.getInlineStyleAt(i).toJS()).toEqual([]);
+            } else {
+                expect(block.getInlineStyleAt(i).toJS()).toEqual(['TOGGLE_BOLD_SUGGESTION-1', 'BOLD']);
+            }
+        }
+
+        block = content.getLastBlock();
+
+        for (let i = 0; i < 10; i++) {
+            if (i < 6) {
+                expect(block.getInlineStyleAt(i).toJS()).toEqual(['TOGGLE_BOLD_SUGGESTION-1', 'BOLD']);
+            } else {
+                expect(block.getInlineStyleAt(i).toJS()).toEqual([]);
+            }
+        }
+
+        expect(Highlights.getHighlightData(editorState, 'TOGGLE_BOLD_SUGGESTION-1')).toEqual({
+            date: date,
+            author: 'author_id',
+            type: 'TOGGLE_BOLD_SUGGESTION',
+            originalStyle: '',
+        });
+    });
+
+    it('CREATE_CHANGE_STYLE_SUGGESTION - set BOLD suggestion for a bold text', () => {
```

Comment:
When creating a test case, describe the expected outcome. Write the "it" case so it's gramatically correct when you read it including "it".

`it('Should set BOLD suggestion even if the text is already bold')` or
`it('Should ignore BOLD suggestion if the text is already bold')`

The same applies to other tests like "set BOLD suggestion 2 times" where it's not specified what is the expected behaviour after setting it 2 times.